locale/en/blog/module/multi-server-continuous-deployment-with-fleet.md

```diff
@@ -89,4 +89,4 @@ For help about a command, try `fleet help `.</pre>
 
 <p><span class="code">npm install -g fleet</span> and <a href="https://github.com/substack/fleet">check out the code on github</a>! </p>
 
-<img src="http://substack.net/images/fleet.png" width="849" height="568">
+<img src="http://substack.net/images/fleet.png" alt="" width="849" height="568">
```

Comment:
This would be enough for the alt text. The URL returns 404 now, but [an archive](https://web.archive.org/web/20150608121451/http://substack.net/images/fleet.png) exists.
```suggestion
<img src="http://substack.net/images/fleet.png" alt="fleet" width="849" height="568">
```

Reply:
Yeah I didn't want to bother a lot with this since it's 404 and should be handled later anyway.

Reply:
Blank alt tag is probably better. This image doesn't provide information that's missing without it, so empty `alt` will be less confusing that having the screen reader bleat out "fleet" with no context.